.travis.yml

```diff
@@ -11,7 +11,7 @@ install:
   - sudo apt-get install libglib2.0 libxml2-dev libpcap-dev librrd-dev redis-server libsqlite3-dev libhiredis-dev libgeoip-dev libcurl4-openssl-dev libzmq3-dev git
 
 before_script:
-  - git clone https://github.com/ntop/nDPI.git; cd nDPI; ./autogen.sh; make; cd ..
+  - git clone https://github.com/ntop/nDPI.git; cd nDPI; ./autogen.sh; make; sudo make install; cd ..
```

Comment:
no need to install nDPI as it may conflict with other existing versions. It must stay in the same dir as ntopng sources

Reply:
As nDPI and ntopng are seen as separate projects, I think it needs to support and test the normal "workflow". It also makes things easier to maintain for OS package managers.

The ntopng Makefile should be smart enough to detect the available nDPI versions (as more versions can coexist on the same system) and use the appropriate one.